web/load.js

```diff
@@ -178,6 +178,12 @@
     addStyleFile('https://cdn.traccar.com/js/ol/' + olVersion + '/ol.css');
     addScriptFile('https://cdn.traccar.com/js/ol/' + olVersion + '/ol.js');
 
+    addStyleFile('https://unpkg.com/ol-popup@4.0.0/src/ol-popup.css');
+    addScriptFile('https://unpkg.com/ol-popup@4.0.0');
+
+    addStyleFile('https://cdn.jsdelivr.net/npm/ol-geocoder@latest/dist/ol-geocoder.min.css');
+    addScriptFile('https://unpkg.com/ol-geocoder');
```

Comment:
We'll have to put this on our CDN.

Reply:
This should be ok, both libraries are MIT licensed:

https://github.com/jonataswalker/ol-geocoder
https://github.com/walkermatt/ol-popup

Reply:
Please create a PR and update this PR with CDN links.